lithops/worker/jobrunner.py

```diff
@@ -20,16 +20,18 @@
 import sys
 import pika
 import time
+import psutil
 import pickle
 import logging
 import inspect
 import requests
+import threading
 import traceback
 from pydoc import locate
 from distutils.util import strtobool
 
 from lithops.worker.utils import peak_memory
-
+from lithops.monitor import CPUMonitor
```

Comment:
is this import still correct?

Reply:
It is correct since it is used from the run function to monitor the execution. Do you agree?

Reply:
As for the psutil dependency in the requirements files, it was an oversight on my part. Sorry :).

Reply:
As far as I can see you moved the CPUMonitor from the monitor.py to the worker/utils.py. So I would expect the import to be` from lithops.worker.utils import CPUMonitor`

Reply:
Yes, you're right! :) I have changed it. Sorry!